srcpkgs/iat/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'iat'
 pkgname=iat
 version=0.1.7
-revision=3
+revision=4
 build_style=gnu-configure
-short_desc="A tool for detecting the structure of many types of CD/DVD image"
+short_desc="Tool for detecting the structure of many types of CD/DVD image"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-3"
+license="GPL-3.0-or-later"
```

Comment:
README and all source files is having `either version 2 of the License, or (at your option) any later version.`
Technically, `GPL-3.0-or-later` is a subset of `GPL-2.0-or-later`.

Also
```
 * @ Redistributions of source code must retain the above copyright
 *   notice, this list of conditions and the following disclaimer. 
 *
 * @ Redistributions in binary form must reproduce the above copyright
 *   notice, this list of conditions and the following disclaimer in
 *   the documentation and/or other materials provided with the
 *   distribution. 
```

Reply:
It is, however the license that they ship in COPYING is the GPL 3 so I went with that. You're right though, they made a mistake when picking a license file to include. I'm going to make a custom "GPL-2.0-or-later-with-attribution-requirement" license and then install that. That should put it in compliance with their requirement of reproducing the conditions list.

Reply:
I looked a second time, that's the BSD-2-Clause license text, just with a "this is licensed under the GPL" header slapped on it.